venues/ICLR.cc/2018/Conference/python/programchairs/get-review-info.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+
+"""
+Output review information for all papers to a csv file.
+Paper Num, Title, Area Chair, reviews complete/total, Reviewer1, rating, confidence, Reviewer2...
+
+"""
+
+## Import statements
+import argparse
+from openreview import *
+import csv
+import sys
+sys.path.insert(0, "./..")
+import config
+
+## Handle the arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+parser.add_argument('-f', '--file',help="specify an output file otherwise defaults to info.csv")
+parser.add_argument('--ac',help="specify area chair")
+args = parser.parse_args()
+if args.file == None:
+    args.file = 'info.csv'
+
+## Initialize the client library with username and password
+client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+reviews_total =0
+reviews_complete = 0
+# paper_data[paper_num]['title'] = title
+# paper_data[paper_num]['reviewers']['rating'/'confidence'] = rating
+# paper_data[paper_num]['AC']= Area Chair id
+# paper_data[paper_num]['reviewer_count']= # of assigned reviewers
+# paper_data[paper_num]['review_count']= # of reviews complete
+paper_data = {}
+
+# fill in submission information
+print "Filling in submissions"
+submissions = client.get_notes(invitation=config.SUBMISSION)
```

Comment:
you need to get the blind submissions: config.BLIND_SUBMISSION

Reply:
Why does it need the blind submissions vs the actual submissions? I'm only getting the title so it doesn't matter that way. Is it because we could accidentally show authors and it's safer to deal with the blind submission?

Reply:
No, it is not that. 

I tested using live data and I got deleted papers. It seems that some paper were deleted only the blind submission but the original one still exists.